config.xml

```diff
@@ -55,10 +55,13 @@
     </platform>
     <platform name="ios">
         <edit-config file="*-Info.plist" mode="merge" target="NSCameraUsageDescription">
-            <string>Used to scan barcodes</string>
+            <string>The app would like to use the camera to scan QR codes.</string>
+        </edit-config>
+        <edit-config file="*-Info.plist" mode="merge" target="NSPhotoLibraryUsageDescription">
+            <string>The app would like to access the photo library for tasks.</string>
```

Comment:
It would be better to specify what specific task it is. In this case, I think it is to read the scanned QR code

Reply:
Okay made the changes.